integrations/chatwoot/src/channels.ts

```diff
@@ -0,0 +1,125 @@
+import * as bp from '.botpress'
+import { RuntimeError } from '@botpress/sdk'
+import { sendMessage, sendAttachment, getApiAccessToken } from './client'
+import { getAccountId } from './actions/hitl'
+
+type ConversationWithTags = { tags: { id?: string } }
+
+const getConversationContext = async (client: bp.Client, ctx: bp.Context, conversation: ConversationWithTags) => {
+  const chatwootConvId = conversation.tags.id
+  if (!chatwootConvId) throw new RuntimeError('No Chatwoot conversation ID')
+  const accountId = await getAccountId(client, ctx)
+  return { chatwootConvId, accountId }
+}
+
+const unsupportedHandler =
+  (type: string) =>
+  async ({ logger }: { logger: bp.Logger }) => {
+    logger.forBot().warn(`${type} messages not supported for Chatwoot`)
```

Comment:
question: is this how we do it in other integrations? I wonder if we should send a message saying something like "unsupported message type", so the build is not too confused.